.github/workflows/tests.yaml

```diff
@@ -744,6 +779,7 @@ jobs:
           copy-connections: false
           copy-airflow-variables: false
           copy-pools: false
+          wake-on-deploy: true
```

Comment:
for test completeness, should we validate that the deployment is not hibernating at the end of deploy?

Reply:
But it can be hibernating and that is okay. It should only be non-hibernating during deploy which is guaranteed because otherwise, the deploy would fail.

But what we could and should do is assert that the hibernation spec override definition is not set.

Reply:
yeah that makes sense, to validate that the hibernation spec is intact after the deploy